.github/workflows/release-firmware-release-config.yml

```diff
@@ -0,0 +1,46 @@
+name: "[Release] firmware release config"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-message-release:
+    if: ${{ github.repository == 'trezor/trezor-suite' && (github.ref == 'refs/heads/release-firmware-release-config-production' || github.ref == 'refs/heads/release-firmware-release-config-develop') }}
+    environment: ${{ github.ref == 'refs/heads/release-firmware-release-config-production' && 'production-firmware-release-config' || 'develop-firmware-release-config' }}
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_ENV: ${{ github.ref == 'refs/heads/release-firmware-release-config-production' && 'production' || 'develop' }}
+      AWS_REGION: "eu-central-1"
+      AWS_CLOUDFRONT_ID: E1ERY5K2OTKKI1
+      ROLE_TO_ASSUME: ${{ github.ref == 'refs/heads/release-firmware-release-config-production' && 'arn:aws:iam::538326561891:role/gh_actions_suite_production_firmware_release_config' || 'arn:aws:iam::538326561891:role/gh_actions_suite_develop_firmware_release_config' }}
```

Comment:
We can keep the names like this, I'll create the roles on AWS tomorrow with this name.